packages/edit-site/src/components/page-pages/index.js

```diff
@@ -111,13 +129,13 @@ export default function PagePages() {
 				id: 'featured-image',
 				header: __( 'Featured Image' ),
 				getValue: ( { item } ) => item.featured_media,
-				render: ( { item, view: currentView } ) =>
+				render: ( { item, view: iterationView } ) =>
```

Comment:
You renamed this because of the new `currentView` declaration above, right? I think we should leave this as was and rename the other one `activeView`. The url param feels better as `activeView`, no?